src/vmm/src/guest_config/aarch/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::guest_config::Modifier;
+
+/// Template providing bitmask modifier info
+/// for CPU register configuration.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct ArmCpuTemplate {
+    /// Map of Modifiers for CPU configuration
+    /// Key: Register pointer
+    /// Value: Register's bitmask based modifier
+    pub modifiers: HashMap<u64, Modifier>,
```

Comment:
Maybe `registers_modifiers`?

Reply:
There have been numerous opinions on long versus short naming. I am not going to make further changes in this regard.